test/utils/stubs/CustomsDeclarationsInformationDownstreamService.scala

```diff
@@ -21,14 +21,14 @@ import com.github.tomakehurst.wiremock.stubbing.StubMapping
 import play.api.http.ContentTypes
 import play.api.mvc.Codec
 import play.api.test.Helpers._
-import stubs.CustomsDeclarationsInformationAPIService._
+import stubs.CustomsDeclarationsInformationDownstreamService._
 import testdata.declarationinformation.DeclarationStatusTestData._
 
 import scala.xml.Elem
 
-trait CustomsDeclarationsInformationAPIService extends WireMockRunner {
+trait CustomsDeclarationsInformationDownstreamService extends WireMockRunner {
```

Comment:
It seems to me that adding `Downstream` to the name is superficial and creates noise. As far as I understand, all services used in this context are 'downstream'. And I don't think we'd lose any relevant information if we had not used this word at all.

Would it be also worth your time to make this trait extend `MockGenericDownstreamService` ?

Reply:
:+1: